salesagility/SuiteCRM
# Add missing language definitions for the module loader

## Description




During installation of a module, there is missing text in the UI and warnings in the logs for undefined indexes

## Motivation and Context

Remove the warnings from the logs so we see real warnings and errors as well as displaying in the UI what is going on.

## How To Test This

* Install a module through module loader
* Check logs
* Look at screen to see difference in displayed text

## Types of changes


### Final checklist
